diff --git a/scripts/coder_format/coder_format.inc b/scripts/coder_format/coder_format.inc
index 9eab367..d508c02 100644
--- a/scripts/coder_format/coder_format.inc
+++ b/scripts/coder_format/coder_format.inc
@@ -638,7 +638,7 @@ function coder_format_string($code = '') {
 
         case T_CONST:
           // Constants are written uppercase.
-          $result = rtrim($result) . strtoupper(trim($text));
+          $result = rtrim($result) . "\n  " . strtoupper(rtrim($text)) . ' ';
           break;
 
         case T_ENCAPSED_AND_WHITESPACE:
